static/docs/user-guide/dvc-file-format.md

```diff
@@ -60,3 +64,11 @@ A metric entry consists of such fields:
 - `type`: type of the metrics file (e.g. raw/json/tsv/htsv/csv/hcsv);
 - `xpath`: path within the metrics file to the metrics data(e.g. `AUC.value` for
   `{"AUC": {"value": 0.624321}}`);
+
+A meta entry consists of `key:value` pairs such as `name: john`. A meta entry
+can have any structure and contain any number of attributes. `"meta: string"` is
+also possible, it doesn't necessarily need to contain dictionary always.
+
+Comments can be added to the .dvc file using `# comment` syntax. Comments
```

Comment:
same here, let's mention that dvc run/add/import when they overwrite/update the file information is lost